backend/corpora/common/authorizer.py

```diff
@@ -2,13 +2,20 @@
 from functools import lru_cache
 
 import requests
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util import Retry
 from jose import jwt
 from jose.exceptions import ExpiredSignatureError, JWTError, JWTClaimsError
 
 from .corpora_config import CorporaAuthConfig
 from backend.corpora.common.utils.http_exceptions import UnauthorizedError
 
 
+auth0_session_with_retry = requests.Session()
+retry_config = Retry(total=3, backoff_factor=1, status_forcelist=[429, 500, 502, 503, 504])
```

Comment:
why are we specifying `status_forcelist` twice? Both here and in the `retry_config`

Reply:
I can consolidate